src/renderer/ModalsLayer.js

```diff
@@ -29,9 +29,9 @@ const BackDrop: ThemedComponent<{ state: string }> = styled.div.attrs(({ state }
 `;
 
 const ModalsLayer = ({ visibleModals }: *) => {
-  const filteredModals = visibleModals.filter(
-    ({ name, MODAL_SHOW_ONCE }) => !MODAL_SHOW_ONCE || !global.sessionStorage.getItem(name),
-  );
+  const filteredModals = visibleModals
+    .filter(({ name, MODAL_SHOW_ONCE }) => !MODAL_SHOW_ONCE || !global.sessionStorage.getItem(name))
+    .slice(0, 1);
```

Comment:
beware this logic appears to prevent any "wrapped" modal to happen. Are we sure there are not a single case where this would be the case in Live?

Reply:
Looking for calls to `openModal(` inside the `/modals` folder would reveal nested modals handled by the modals layer. I went over all the matches (7) and the only case that seem to show a stacked modal is:
- Account migration, which we don't currently use https://github.com/LedgerHQ/ledger-live-desktop/blob/master/src/renderer/modals/MigrateAccounts/steps/StepOverview.js#L177 
All other usages are after a `closeModal` call, preventing the stack.